magefile.go

```diff
@@ -65,26 +65,53 @@ func Build() error {
 }
 
 func fetchPackageStorage() error {
-	err := os.RemoveAll(storageRepoDir)
+
+	// If storage directory does not exists, check it out
+	if _, err := os.Stat(storageRepoDir); os.IsNotExist(err) {
+		err = sh.Run("git", "clone", "--depth=1", "https://github.com/elastic/package-storage.git", storageRepoDir)
+		if err != nil {
+			return err
+		}
+	} else if err != nil {
+		// catch other errors
+		return err
+	}
```

Comment:
I suggest to not play with Git after cloning and simply return here.

Reply:
Agree, adjusted.